embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/spans/SpansServiceImpl.kt

```diff
@@ -98,7 +98,6 @@ internal class SpansServiceImpl(
     override fun getSpan(spanId: String): EmbraceSpan? = spansRepository.getSpan(spanId = spanId)
 
     companion object {
-        const val MAX_TRACE_COUNT_PER_SESSION = 100
-        const val MAX_SPAN_COUNT_PER_TRACE = 10
+        const val MAX_NON_INTERNAL_SPANS_PER_SESSION = 500
```

Comment:
We should tell iOS about this limit. Also, do we need similar limits in future for SpanEvent & attributes?

Reply:
iOS should be aware of the new limits and have a task to implement that in the new version.

We also have existing limits in `EmbraceSpanEvent` and `EmbraceSpanImpl`. We may want to consolidate them in a common location for clarity. I can do that in another PR.